tests/test_helpers/getters_helpers.ts

```diff
@@ -33,9 +33,6 @@ export function getCell(
   return model.getters.getCell(sheetId, col, row);
 }
 
-/**
- * Get the cell at the given XC
- */
```

Comment:
Removed Js doc ? :eyes:

Reply:
its rather useless IMO.
The API is self-explanatory.
I didn't want to add such a docstring for the new `getEvaluatedCell`. So for consistency, I removed this one